src/cases/components/case-hearings-list/case-hearings-list.component.ts

```diff
@@ -1,11 +1,11 @@
-import { Component, Input, OnInit } from '@angular/core';
-import { ActivatedRoute, Router } from '@angular/router';
-import { Store } from '@ngrx/store';
-import { Observable } from 'rxjs';
-import { LovRefDataModel } from '../../../hearings/models/lovRefData.model';
-import { HearingConditions } from '../../../hearings/models/hearingConditions';
-import { HearingListViewModel } from '../../../hearings/models/hearingListView.model';
-import { Actions, EXUIDisplayStatusEnum, EXUISectionStatusEnum, Mode } from '../../../hearings/models/hearings.enum';
+import {Component, Input, OnInit} from '@angular/core';
+import {ActivatedRoute, Router} from '@angular/router';
+import {Store} from '@ngrx/store';
+import {Observable} from 'rxjs';
+import {HearingConditions} from '../../../hearings/models/hearingConditions';
```

Comment:
its more of check, we can configure some path inside tsconfig.json to go with relative path instead of absolute, later on i guess, like this,
        "paths": {
          "@hearingsModels/*": ["../../../hearings/models"],
          "@shared/*": ["shared/*"],
        },

Reply:
that is not neccessary to add this to the tsconfig path. If you are using the right plugins in the IDE, the relative path is automatically imported, and I have never written this relative path myself.